src/tools/convert/topology_helper.py

```diff
@@ -58,7 +58,12 @@ def FillFileBuffer(fname, filebuffer):
     for line in f:
 	if "include" in line and not line[0]==';':
 	    name=(line.split()[1]).strip('\"')
-	    FillFileBuffer(name, filebuffer)
+            try: 
+ 	        FillFileBuffer(name, filebuffer)
+            except IOError:
+                #need to use relative path
+                name = fname.split('/')[0] + '/' + name
```

Comment:
Okay, got it.

The current version of topology_helper.py in this pull request does work with the line 
#include "/usr/share/gromacs/top/opls.ff/forcefield.itp"
in topol.top, including when forcefield.itp then includes other files whose path is given relative to `/usr/share/gromacs/top/opls.ff/`. And it also works for the various other cases I described above.

It doesn't work for `#define` statements, but that's always been the case. That would also require changes in gromacs.py. And it's not relevant to this pull request. (Actually I never use `#define` statements and don't know how they work :) )